crates/bevy_core_pipeline/src/oit/resolve/mod.rs

```diff
@@ -61,6 +64,17 @@ impl Plugin for OitResolvePlugin {
             return;
         }
 
+        if render_app
+            .world()
+            .resource::<RenderDevice>()
+            .limits()
+            .max_storage_buffers_per_shader_stage
+            < OIT_REQUIRED_STORAGE_BUFFERS
+        {
+            warn!("OrderIndependentTransparencyPlugin not loaded. RenderDevice lacks support: max_storage_buffers_per_shader_stage < OIT_REQUIRED_STORAGE_BUFFERS.");
```

Comment:
```suggestion
            warn!("OrderIndependentTransparencyPlugin not loaded. RenderDevice lacks support: max_storage_buffers_per_shader_stage < {OIT_REQUIRED_STORAGE_BUFFERS}.");
```

I think it would be better to just write the value. Or maybe add it after in parentheses?

Reply:
Trying a fancier version in c19f0f2855c94c6a9b1114f94e6f47d503d22c68. Maybe a little too wordy now.

```
OrderIndependentTransparencyPlugin not loaded. RenderDevice lacks support: max_storage_buffers_per_shader_stage < OIT_REQUIRED_STORAGE_BUFFERS. max_storage_buffers_per_shader_stage=0 OIT_REQUIRED_STORAGE_BUFFERS=2
```